src/lightning_app/core/app.py

```diff
@@ -46,6 +48,7 @@ def __init__(
         self,
         root: "lightning_app.LightningFlow",
         debug: bool = False,
+        info: add_meta.AppInfo = None,
```

Comment:
This argument is missing in the docstring.

Reply:
yeah @pritamsoni-hsr can you add `into` to the arguments on line 68

Reply:
yeah I added this now and marked this in PR description.